mappings/net/minecraft/client/gl/GlExtensionHandler.mapping

```diff
@@ -59,59 +59,138 @@ CLASS none/cbg net/minecraft/client/gl/GlExtensionHandler
+		ARG 1 internalFormat
+		ARG 2 width
+		ARG 3 height
 	METHOD a framebufferTexture2D (IIIII)V
+		ARG 0 target
+		ARG 1 attachment
+		ARG 2 textarget
+		ARG 3 texture
+		ARG 4 level
 	METHOD a getUniformLocation (ILjava/lang/CharSequence;)I
+		ARG 0 program
+		ARG 1 name
 	METHOD a shaderSource (ILjava/nio/ByteBuffer;)V
+		ARG 0 shader
+		ARG 1 source
 	METHOD a bufferData (ILjava/nio/ByteBuffer;I)V
 		ARG 0 target
 		ARG 1 data
 		ARG 2 usage
 	METHOD a uniform1fv (ILjava/nio/FloatBuffer;)V
+		ARG 0 location
+		ARG 1 values
 	METHOD a uniform1iv (ILjava/nio/IntBuffer;)V
+		ARG 0 location
+		ARG 1 values
 	METHOD a uniformMatrix2fv (IZLjava/nio/FloatBuffer;)V
+		ARG 0 location
+		ARG 1 transpose
+		ARG 2 matrices
 	METHOD a shellExecute (Ljava/io/File;)V
 	METHOD b canUseShaders ()Z
 	METHOD b createShader (I)I
+		ARG 0 type
 	METHOD b attachShader (II)V
+		ARG 0 program
+		ARG 1 shader
 	METHOD b framebufferRenderbuffer (IIII)V
+		ARG 0 target
+		ARG 1 attachment
+		ARG 2 renderbufferTarget
+		ARG 3 renderbuffer
 	METHOD b getAttribLocation (ILjava/lang/CharSequence;)I
+		ARG 0 program
+		ARG 1 name
 	METHOD b uniform2fv (ILjava/nio/FloatBuffer;)V
+		ARG 0 location
+		ARG 1 values
 	METHOD b uniform2iv (ILjava/nio/IntBuffer;)V
+		ARG 0 location
+		ARG 1 values
 	METHOD b uniformMatrix3fv (IZLjava/nio/FloatBuffer;)V
+		ARG 0 location
+		ARG 1 transpose
+		ARG 2 matrices
 	METHOD c getInitInfoLog ()Ljava/lang/String;
 	METHOD c compileShader (I)V
+		ARG 0 shader
 	METHOD c getShader (II)I
+		ARG 0 shader
+		ARG 1 pname
 	METHOD c blendFuncSeparate (IIII)V
+		ARG 0 sFactorRGB
+		ARG 1 dFactorRGB
+		ARG 2 sFactorAlpha
+		ARG 3 dFactorAlpha
 	METHOD c uniform3fv (ILjava/nio/FloatBuffer;)V
+		ARG 0 location
+		ARG 1 values
 	METHOD c uniform3iv (ILjava/nio/IntBuffer;)V
+		ARG 0 location
+		ARG 1 values
 	METHOD c uniformMatrix4fv (IZLjava/nio/FloatBuffer;)V
+		ARG 0 location
+		ARG 1 transpose
+		ARG 2 matrices
 	METHOD d createProgram ()I
 	METHOD d useProgram (I)V
+		ARG 0 program
 	METHOD d getShaderInfoLog (II)Ljava/lang/String;
+		ARG 0 shader
+		ARG 1 maxLength
 	METHOD d uniform4fv (ILjava/nio/FloatBuffer;)V
+		ARG 0 location
+		ARG 1 values
 	METHOD d uniform4iv (ILjava/nio/IntBuffer;)V
+		ARG 0 location
+		ARG 1 values
 	METHOD e genBuffers ()I
 	METHOD e deleteProgram (I)V
+		ARG 0 program
 	METHOD e getProgramInfoLog (II)Ljava/lang/String;
+		ARG 0 program
+		ARG 1 maxLength
 	METHOD f useVertexBufferObjects ()Z
 	METHOD f linkProgram (I)V
+		ARG 0 program
 	METHOD f uniform1i (II)V
+		ARG 0 location
+		ARG 1 value
 	METHOD g genFramebuffers ()I
 	METHOD g deleteBuffers (I)V
+		ARG 0 buffer
 	METHOD g bindBuffer (II)V
 		ARG 0 target
 		ARG 1 bufferId
 	METHOD h genRenderbuffers ()I
 	METHOD h deleteRenderbuffers (I)V
+		ARG 0 renderbuffer
 	METHOD h bindFramebuffer (II)V
+		ARG 0 target
+		ARG 1 framebuffer
 	METHOD i deleteFramebuffers (I)V
+		ARG 0 framebuffer
 	METHOD i bindRenderbuffer (II)V
+		ARG 0 target
+		ARG 1 renderbuffer
```

Comment:
A bunch more buffers here that should probably be `renderBuffer`/`frameBuffer` ..?